build.gradle

```diff
@@ -396,6 +396,7 @@ run {
             '--add-exports', 'org.controlsfx.controls/impl.org.controlsfx.skin=org.jabref',
             '--add-opens', 'javafx.controls/javafx.scene.control=org.jabref',
             '--add-opens', 'org.controlsfx.controls/org.controlsfx.control.textfield=org.jabref',
+            '--add-opens', 'javafx.controls/com.sun.javafx.scene.control=org.jabref',
```

Comment:
Yeah, I was about to report that. Encountered this yesterday but forgot to report it